ProcessMaker/processmaker
# FOUR-11118 Compatibility with validation messages for testing

## Issue & Reproduction Steps
Prior to the Laravel update, field names in validation messages were capitalized, which our automated tests rely on for accuracy. The recent update may have altered this capitalization convention, potentially affecting test outcomes.

## Solution
- Capitalize the field name in validation messages

## How to Test
- Create a Decision Table
- Try to save it without name
- Validation must be: `The Name field is required.`
- Note that `Name` must be capitalized

## Related Tickets & Packages
- https://processmaker.atlassian.net/browse/FOUR-11118

## Code Review Checklist

ci:next

.
